test/math/FloatVectorListTest.ooc

```diff
@@ -293,6 +293,54 @@ FloatVectorListTest: class extends Fixture {
 			list free()
 			filtered free()
 		})
+		this add("linear interpolation for list", func {
+			list := FloatVectorList new()
+			list add(2.0f)
+			list add(1.0f)
+			list add(6.0f)
+			list add(4.0f)
+			list add(7.0f)
+			numberOfPointsBetween := 1
+			tempLists := list interpolate(numberOfPointsBetween)
+
+			expect(tempLists count, is equal to(list count + numberOfPointsBetween * (list count - 1)))
+			expect(tempLists[0], is equal to(2.0f) within(tolerance))
+			expect(tempLists[1], is equal to(1.5f) within(tolerance))
+			expect(tempLists[2], is equal to(1.0f) within(tolerance))
+			expect(tempLists[3], is equal to(3.5f) within(tolerance))
+			list free()
+			tempLists free()
+		})
+		this add("find CrossCorrelation Offset", func {
+			list := FloatVectorList new()
+			list add(2.0f)
+			list add(1.0f)
+			list add(6.0f)
+			list add(4.0f)
+			list add(7.0f)
+			list add(6.0f)
+			list add(6.0f)
+			list add(4.0f)
+			list add(7.0f)
+			list add(4.0f)
+			list add(7.0f)
+			tempLists0 := list shift(2)
+			tempLists1 := list shift(-1)
+			tempLists2 := list shift(0)
+
+			temp0 := list findCrossCorrelationOffset(tempLists0, -3, 3)
+			temp1 := list findCrossCorrelationOffset(tempLists1, -3, 3)
```

Comment:
tempLists should probably be called shiftedList or similar, and temp0 called correlationOffset or something similar. A name that implies what the variable contains, so that someone reading this doesn't have to read the previous code to understand what happens later. Same with the first test.